### ISSUE TITLE
Comments on Section 4.2

### ISSUE THREAD

**Issue by acbegen** [Dash-Industry-Forum/Events#59]
Assume someone has a long timeshiftbuffer. Media samples as well as the events/timed metadata samples are buffered per Section 4.2. Does that mean if the content is played back again, the same events will be triggered? I suppose yes, but is there a way to disable this like triggering an event only once or twice for example, not every time the corresponding media is played again?

**Comment by irajs** [comment]
No there is no way to disable triggering back events that were triggered before? we need some sort of client session definition to define a state for it.

for instance, if a portion of media presentation is played, then it was stopped and then played again, 10 mins apart, is that one session or two?

**Comment by acbegen** [comment]
there could be another way. Maybe add an "expiration date/time" on the events? If it is not specified, it will trigger every time, it will be optional.

**Comment by haudiobe** [comment]
(Event TF - 19/10/04) This is not part of this version. New signaling would need much more discussion. Also consider that this is an application logic to ignore Events that are sent duplicated. We should not overload the Event mechanism.
